docs/quickstart.md

```diff
@@ -200,12 +200,12 @@ $ scp -r -i data/id_rsa root@158.175.161.118:~/openstack-upi/auth/\* .
 
 OpenShift CLI `oc` can be downloaded from the following links. Use the one specific to your client system architecture.
 
-- [Mac OSX](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/macosx/oc.tar.gz)
-- [Linux (x86_64)](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/linux/oc.tar.gz)
-- [Windows](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/windows/oc.zip)
+- [Mac OSX](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/latest/macosx/oc.tar.gz)
```

Comment:
Just mentioning here again.. these oc links are deprecated.

Reply:
Yes, thanks for pointing out. I'll update it